e2e/test-runner/runner_test.go

```diff
@@ -1,35 +1,162 @@
 package test_runner
 
 import (
+	"gopkg.in/ini.v1"
 	"strings"
 	"testing"
+	"time"
 )
 
+func expectedUpgradePolicy() string {
+	if *distroName == "Ubuntu" {
+		return "lts"
+	}
+	if strings.HasPrefix(*distroName, "Ubuntu") && strings.HasSuffix(*distroName, "LTS") {
+		return "never"
+	}
+	// Preview and Dev
+	return "normal"
+}
+
 func TestBasicSetup(t *testing.T) {
 	// Wrapping testing.T to get the friendly additions to its API as declared in `wsl_tester.go`.
 	tester := WslTester(t)
 
 	// Invoke the launcher as if the user did: `<launcher>.exe install --ui=gui`
-	outStr := tester.AssertLauncherCommand("install --ui=gui") // without install the launcher will run bash.
+	outStr := tester.AssertLauncherCommand("install --root --ui=none") // without install the launcher will run bash.

```

Comment:
```suggestion
	outStr := tester.AssertLauncherCommand("install --ui=gui") // without install the launcher will run bash.
```
This debug code slipped through :(